cli/lib/exec/xvfb.js

```diff
@@ -9,6 +9,7 @@ const util = require('../util')
 
 const xvfb = Promise.promisifyAll(new Xvfb({
   timeout: 30000, // milliseconds
+  xvfb_args: ['-screen', '0', '1280x1024x24'], // need to explicitly define screen otherwise electron will crash
```

Comment:
I took this value from [freedesktop/xorg-xserver](https://github.com/freedesktop/xorg-xserver/blob/master/hw/vfb/InitOutput.c#L277) but there has to be something rotten with ubuntu's xvfb, because it's mentioned in man as well as in [code](https://github.com/freedesktop/xorg-xserver/blob/2a2234ad1a0fe88400c1511fea67741e4ad09f7f/hw/vfb/InitOutput.c#L273) that `-screen` is default to `1280x1024x24` 🤔

Reply:
Actualyl ubuntu uses this packag [bionic/xvfb](https://packages.ubuntu.com/bionic/xvfb) contains older version with: 

```c
#define VFB_DEFAULT_WIDTH      1280
#define VFB_DEFAULT_HEIGHT     1024
#define VFB_DEFAULT_DEPTH         8
```

So I thought that Electron is freaking out because of the bit depth, but when I start Xvbf with 8bits electron is still fine with that even without `-screen` at all 🤷‍♂

Reply:
So latest development indicates that it could be problem somewhere else and this is just red herring:

```console
[2031:0121/105922.460578:FATAL:memory.cc(22)] Out of memory. size=65536
We detected that the Chromium Renderer process just crashed.

This is the equivalent to seeing the 'sad face' when Chrome dies.
(...)
```